### PR TITLE
upgrade jindocache to 6.2.0 and support forceload

### PR DESCRIPTION
**Pull request by frankleaf**
### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/fluid-dataloader/jindocache/CHANGELOG.md [hunk 1]
```diff
@@ -18,7 +20,7 @@
 ### 0.2.0
 - Support JindoFSx
 
-### 0.1.0
+### 0.2.0
```
Comment (cheyang) [inline charts/fluid-dataloader/jindocache/CHANGELOG.md:23]: Why do you need to change 0.1.0  to 0.2.0?

Comment (frankleaf) [reply charts/fluid-dataloader/jindocache/CHANGELOG.md]: Accidentally modified，done

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**